components/projects/upload/ProjectServiceType.tsx

```diff
@@ -10,24 +11,30 @@ import { MOBILE_MEDIA_QUERY } from '@/styles/mediaQuery';
 import { textStyles } from '@/styles/typography';
 
 const ProjectServiceType = () => {
-  const { watch, register } = useFormContext<ProjectUploadForm>();
+  const {
+    watch,
+    register,
+    formState: { errors },
+  } = useFormContext<ProjectUploadForm>();
   const selectedServices = watch('serviceType');
 
   return (
     <StyledContainer>
       <FormTitle essential description='복수 선택 가능'>
         서비스 형태
       </FormTitle>
-      <StyledContent>
-        <StyledLabel checked={selectedServices.includes(ServiceType.WEB)}>
-          <input type='checkbox' value={ServiceType.WEB} {...register('serviceType')} />
-          <span>웹</span>
-        </StyledLabel>
-        <StyledLabel checked={selectedServices.includes(ServiceType.APP)}>
-          <input type='checkbox' value={ServiceType.APP} {...register('serviceType')} />
-          <span>앱</span>
-        </StyledLabel>
-      </StyledContent>
+      <FormItem errorMessage={errors.serviceType?.message}>
```

Comment:
지금 에러나는 이유
```suggestion
      <FormItem errorMessage={errors.serviceType?.[index].message}>
```

Field의 index를 prop으로 넘겨주든지 해서 이렇게 해줘야 해용
`errors.serviceType`이 `FieldError` 타입이 아니라 `FieldError[]` 타입이어서 뜨는 에러!

Reply:
<img width="1040" alt="image" src="https://user-images.githubusercontent.com/73823388/202880085-61b1ba33-6973-4fa6-b0ae-3d837c413e97.png">

actions에서도 이렇게 뜨는데 @juno7803 환경에서는 이상하게 이게 안 잡혔나 . .

Reply:
실행해보니까 이렇게 타입 에러 나는 상태에서도 UI에 에러메시지는 잘 뜨네요 이상하다 ..
그리고 그냥 `errors.serviceType?.[0].message` 이렇게만 해줘도 잘 동작해요

Reply:
오오오 맞네 저 타입이 배열이라서 0번째 인덱스 그대로 읽어도 될것 같다.. 고마워 ㅠㅡㅠ
저렇게 수정할게요!!!

Reply:
웅 마자 사실 액션에서는 확인했는데 vscode나 실제 동작은 잘 이뤄지고 있어서 (타입도 라이브러리 정의대로 잘 가져온거라..)도대체 뭐지 했었어.. `ServiceType`이 배열인걸 잊고 있었당 ㅎㅎ,, 왜 `useFieldArray`도 아닌데 저럴까 하고만 있었거든.. 고마워!!

f53bb4fa94bdaa28351118a34a8dabcab3d4eeba 요 커밋으로 수정했슴당

Reply:
@NamJwong 오 근데 배열로 접근하면 안뜬다..? 실제로 확인해도 요거구.. 아예 그냥 ts에러 안잡히게 해둬야 할지도 모르겠다.. 일단 라이브러리 업뎃한번 해보던지 할겡
<img width="685" alt="image" src="https://user-images.githubusercontent.com/26808056/202893136-fb9f9864-bce2-471f-ac20-23e1975a16c1.png">